src/routes/applicant.ts

```diff
@@ -0,0 +1,15 @@
+import * as express from 'express';
+import { createApplicant, updateApplicantStatus, listAllApplicants } from '../controllers/applicant';
+
+const router = express.Router();
+
+/* Returns a list of all applicants currently in the database */
+router.get('/', listAllApplicants);
+
+/* Turns the JSON object passed in the request body into a new applicant */
+router.post('/create', createApplicant);
+
+/* Sets the status of the applicant matching 'id' */
+router.patch('/:id/status', updateApplicantStatus);
```

Comment:
Just missed the RESTful API comment from previous review - please fix and then looks good.